### PR TITLE
Upgrate node to 8 LTS and installing m2-builder npm package

### PR DESCRIPTION
**Pull request by dcabrejas**
These tools are needed for running Shane's FE performance process on CircleCI.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y \
     php7.0-pdo-mysql
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-RUN npm install -g yarn
+RUN npm install -g yarn m2-builder@1.0.6
```
Comment (shakyShane) [inline Dockerfile]: @dcabrejas can you ensure this is up to date with latest version seen here https://www.npmjs.com/package/m2-builder

Comment (shakyShane) [reply Dockerfile]: 1.2.0 in this case

Comment (mikeymike) [reply Dockerfile]: is there not a ^1.0 option we can do with NPM ?

Comment (shakyShane) [reply Dockerfile]: @mikeymike  `m2-builder@1`

**Pull request merged**